docs/guides/q-ctrl-performance-management.ipynb

```diff
@@ -176,325 +115,451 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The result of the submitted job had 1 PUB and has a value:\n",
+      " PrimitiveResult([PubResult(data=DataBin(evs=np.ndarray(<shape=(1,), dtype=float64>), stds=np.ndarray(<shape=(1,), dtype=float64>)), metadata={'precision': None})], metadata={})\n",
+      "\n",
+      "The associated PubResult of this job has the following DataBins:\n",
+      " DataBin(evs=np.ndarray(<shape=(1,), dtype=float64>), stds=np.ndarray(<shape=(1,), dtype=float64>))\n",
+      "\n",
+      "And this DataBin has attributes: dict_keys(['evs', 'stds'])\n",
+      "The expectation values measured from this PUB are: \n",
+      "[-0.01464844]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\n",
+    "    f\"The result of the submitted job had {len(result)} PUB and has a value:\\n {result}\\n\"\n",
+    ")\n",
+    "print(\n",
+    "    f\"The associated PubResult of this job has the following DataBins:\\n {result[0].data}\\n\"\n",
+    ")\n",
+    "print(f\"And this DataBin has attributes: {result[0].data.keys()}\")\n",
+    "print(f\"The expectation values measured from this PUB are: \\n{result[0].data.evs}\")"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "df2788d5",
+   "id": "20cb6ab1",
    "metadata": {},
    "source": [
-    "## Example: Estimator\n",
+    "### Estimator inputs\n",
+    "<Admonition type=\"caution\">Fire Opal Performance Management accepts abstract circuits, in contrast to the native Qiskit Runtime primitives, which only accept circuits that are written in the target backend’s Instruction Set Architecture (ISA). For best results, do not transpile circuits before submitting via the Performance Management function. </Admonition>\n",
     "\n",
-    "Use the Sampler mode of Fire Opal Performance Management to run a Bernstein–Vazirani circuit. This algorithm, used to find a hidden string from the outputs of a black box function, is a common benchmarking algorithm because there is a single correct answer."
+    "| Name  | Type  | Description | Required | Default  | Example |\n",
+    "|------|------|----------|----------|-----------|-----------|\n",
+    "| pubs       | `EstimatorPubLike` or `list[EstimatorPubLike]` | One or more tuples containing the inputs listed under `EstimatorPubLike` components   | Yes   | N/A   | `(circuit, observables, parameter_values)`   |\n",
+    "| instance   | `str`   | The hub/group/project to use in that format  | No  | A Premium access instance is randomly chosen if your account has access to multiple instances | `\"hub1/group1/project1\"`     |\n",
+    "| backend_name| `str`  | The name of the backend | No  |  The least busy backend that your instance has access to | `\"ibm_fez\"` |\n",
+    "| options| `dict`  | Input options; see Options section for more details | No  |  See the Options section for details | `{\"default_shots\": 2048}` |\n",
+    "\n",
+    "**`EstimatorPubLike` components:**\n",
```

Comment:
I think this should also point to [Primitive input and output](https://docs.quantum.ibm.com/guides/primitive-input-output)